README.md

```diff
@@ -98,10 +97,10 @@ and Drop component to a lesson, then click the `EDIT` button.
 ![Edit view](https://raw.githubusercontent.com/edx-solutions/xblock-drag-and-drop-v2/c955a38dc3a1aaf609c586d293ce19b282e11ffd/doc/img/edit-view.png)
```

Comment:
@e-kolpakov: there are a few other images in that file that are referenced this way; do you want to update the rest of them as well? That way, we can set a baseline for the rest of the commits on this feature and not have to visit it again.